src/docs/implementations/dynatrace.adoc

```diff
@@ -205,14 +255,18 @@ MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTE
 [source,yml]
 ----
 management.metrics.export.dynatrace:
-    # This is required
-    uri: https://your-environment-id.live.dynatrace.com
-
-    # This is required
-    api-token: MY_TOKEN
-
-    # This is required
-    device-id: sample
+    # For v1 export, do not append a path to the endpoint URL, e.g.:
+    # For SaaS: https://{your-environment-id}.live.dynatrace.com
+    # For managed deployments: https://{your-domain}/e/{your-environment-id}
+    uri: ${DT_METRICS_INGEST_URL}
```

Comment:
I'd rather not touch this since the automatically populated env vars would not work here.
```suggestion
    uri: https://your-environment-id.live.dynatrace.com
```